\documentclass[main.tex]{subfiles}


\begin{document}

\section{Methods}
\label{methods}


\subsection{SEARCH-MaP of the 2,924 nt segment of SARS-CoV-2 genomic RNA}

A DNA template of the 2,924 nt segment of the SARS-CoV-2 genome, plus an upstream T7 promoter, was amplified from our previously constructed pmirGLO plasmid \cite{Lan2022} with 250 nM primers TAATACGACTCACTATAGAATAATGAGCTTAGTCCTGTTGCACTACG and TAAATTGCGGACATACTTATCGGCAATTTTGTTACC (Thermo Fisher Scientific) using 2X CloneAmp HiFi PCR Premix (Takara Bio) in a 50 \textmu l volume with initial denaturation at 98\textdegree C for 60 s; 35 cycles of 98\textdegree C for 10 s, 65\textdegree C for 10 s, and 72\textdegree C for 15 s; and final extension at 72\textdegree C for 60 s.
The 50 \textmu l PCR product was mixed with 10 \textmu l of 6X Purple Loading Dye (New England Biolabs) alongside 10 \textmu l of 0.1X 1 kb DNA Ladder (New England Biolabs) and electrophoresed through a 1\% agarose gel -- 50 ml of 1X tris-acetate-EDTA buffer (Boston BioProducts), 0.5 g of SeaKem Agarose (Lonza), and 5 \textmu l of 10,000X SYBR Safe (Invitrogen) -- in 1X tris-acetate-EDTA buffer (Boston BioProducts) at 60 V for 60 min.
The band at roughly 3 kb was excised and the DNA purified using a Zymoclean Gel DNA Recovery Kit (Zymo Research) according to the manufacturer's protocol; samples were eluted in 10 \textmu l of nuclease-free water (Fisher Bioreagents) and measured with a NanoDrop (Thermo Fisher Scientific).
To increase the DNA yield, the gel-extracted DNA was amplified by a second round of PCR followed by gel extraction, using the same protocol as above.
To remove contaminants after the second gel extraction, the DNA was further purified using a DNA Clean \& Concentrator-5 kit (Zymo Research) according to the manufacturer's protocol; samples were eluted in 10 \textmu l of nuclease-free water (Fisher Bioreagents) and measured with a NanoDrop (Thermo Fisher Scientific).

RNA was transcribed using a MEGAscript T7 Transcription Kit (Invitrogen) according to the manufacturer's protocol.
Specifically, 1 \textmu l (150 ng) of DNA template from the previous step was diluted in 7 ul of nuclease-free water (Fisher Bioreagents), mixed with 2 \textmu l of each 10X ribonucleotide solution followed by 2 \textmu l of the 10X reaction buffer and 2 \textmu l of the 10X enzyme mix, and then incubated at 37\textdegree C for 3 hr.
The DNA template was then degraded by adding 1 \textmu l of TURBO DNase (Invitrogen) and incubating at 37\textdegree C for 15 min.
The RNA transcript was purified using an RNA Clean \& Concentrator-5 kit (Zymo Research) according to the manufacturer's protocol; samples were eluted in 20 \textmu l of nuclease-free water (Fisher Bioreagents) and measured with a NanoDrop (Thermo Fisher Scientific).
DID YOU CONFIRM THE RNA HAS NO OFF-TARGET BANDS?

Antisense oligonucleotides (ASOs) were ordered from Integrated DNA Technologies in a 96-well PCR plate, each ASO already resuspended to 10 \textmu M in 1X IDTE buffer (10 mM Tris, 0.1 mM EDTA).
Each ASO pool (of up to 5 ASOs) was made by mixing 2.5 \textmu l (25 pmol) of each constituent ASO (Supplementary Table \ref{asos2924}); the total volume of each pool was adjusted to 12.5 \textmu l by adding TE Buffer: 10 mM Tris (Invitrogen) with 0.1 mM EDTA (Invitrogen).
Each 12.5 \textmu l ASO pool was then mixed with 1 \textmu l (425 ng, 453 fmol) of RNA (55X molar excess of each ASO) in a PCR tube.
The tubes were heated to 95\textdegree C for 60 seconds to denature the RNA, then placed on ice for several minutes.
The RNA was transferred to 1.5 ml tubes; to each, 35 \textmu l of 1.4X refolding buffer comprising 400 mM sodium cacodylate (Electron Microscopy Sciences) and 6 mM magnesium chloride (Invitrogen) was added, followed by incubation at 37\textdegree C for 25 min to allow the RNA to refold and bind the ASOs.
No-ASO control 1 was handled in the same manner but with 12.5 \textmu l of TE Buffer in lieu of an ASO pool.
For no-ASO control 2, 12.5 \textmu l of TE Buffer was added after placing on ice and before adding refolding buffer, to confirm that the timing of adding TE buffer would not alter the RNA structure.

For chemical probing, 1.5 \textmu l of neat DMS (Sigma-Aldrich) was added to each tube for a total volume of 50 \textmu l including 320 mM DMS, 280 mM cacodylate, 4.2 mM magnesium chloride, and 9.1 nM RNA.
DMS was initially mixed by swirling the pipette tip and kept resuspended by shaking at 500 rpm in a thermomixer (Eppendorf) throughout the treatment at 37\textdegree C for 5 min.
Reactions were quenched by adding 30 \textmu l neat beta-mercaptoethanol (Sigma-Aldrich) and mixing thoroughly by pipetting.
Each sample of DMS-modified RNA was purified using an RNA Clean \& Concentrator-5 kit (Zymo Research) according to the manufacturer's protocol; samples were eluted in 10 \textmu l of nuclease-free water (Fisher Bioreagents) and measured with a NanoDrop (Thermo Fisher Scientific).

ASOs were removed from each sample using TURBO DNase (Invitrogen) according to the manufacturer's protocol.
Briefly, 4 \textmu l of each DMS-modified RNA was mixed with 4 \textmu l of nuclease-free water (Fisher Bioreagents), 1 \textmu l of 10X TURBO DNase Buffer, and 1 \textmu l of TURBO DNase in a PCR tube; and then incubated at 37\textdegree C for 30 min.
To stop each reaction, 2 \textmu l of DNase Inactivation Reagent was mixed in and incubated at room temperature for 10 min, and mixed throughout by flicking several times.
The DNase Inactivation Reagent was precipitated by spinning the tubes on a benchtop PCR tube centrifuge for 10 min, then transferring 4 \textmu l of each supernatant to a new tube.

For reverse transcription, each 4 \textmu l sample of DNased, DMS-modified RNA was mixed with 6 \textmu l of nuclease-free water (Fisher Bioreagents), 4 \textmu l of 5X First Strand Buffer (Invitrogen), 1 \textmu l of 10 mM dNTPs (Promega), 1 \textmu l of 100 mM dithiothreitol (Invitrogen), 1 \textmu l of RNaseOUT (Invitrogen), 1 \textmu l of TGIRT-III (InGex), 1 \textmu l of 10 mM FSE reverse primer CTTCGTCCTTTTCTTGGAAGCGACA (Integrated DNA Technologies), and 1 \textmu l of 10 mM section-specific reverse primer (Integrated DNA Technologies, Supplementary Table \ref{primers2924}), for a total volume of 20 \textmu l.
Synthesis of cDNA was performed at 57\textdegree C for 90 min, then inactivated at 85\textdegree C for 15 min.
To remove the RNA template from the cDNA, 1 \textmu l of Hybridase Thermostable RNase H (Lucigen) was added to each tube and incubated at 37\textdegree C for 20 min.

The cDNA products were amplified using the Advantage HF 2 PCR Kit (Takara Bio) according to the manufacturer's protocol.
Specifically, 1 \textmu l of unpurified cDNA was mixed with 8.25 \textmu l of nuclease-free water (Fisher Bioreagents), 1.25 \textmu l of 10X Advantage 2 PCR Buffer, 1.25 \textmu l of 10X Advantage-HF 2 dNTP Mix, 0.25 \textmu l of 50X Advantage-HF 2 Polymerase Mix, 0.25 \textmu l of 10 mM forward primer (Integrated DNA Technologies, Supplementary Table \ref{primers2924}), and 0.25 \textmu l of 10 mM reverse primer (Integrated DNA Technologies, Supplementary Table \ref{primers2924}), for a total volume of 12.5 \textmu l.
After an initial denaturation at 94\textdegree C for 60 s, 25 cycles of 94\textdegree C for 30 s, 60\textdegree C for 30 s, and 68\textdegree C for 60 s were run, followed by a final extension at 68\textdegree C for 60 s.
For each cDNA, the PCR products were pooled (5 \textmu l each) and then purified using a DNA Clean \& Concentrator-5 kit (Zymo Research) according to the manufacturer's protocol; samples were eluted in 20 \textmu l of nuclease-free water (Fisher Bioreagents) and measured with a NanoDrop (Thermo Fisher Scientific).

A 200 ng aliquot of each pool of PCR products was diluted in 10 mM Tris-HCl, pH 8 (Invitrogen) to a total of 50 \textmu l.
Aliquots were prepared for sequencing using the NEBNext Ultra II DNA Library Prep Kit for Illumina (New England Biolabs) according to the manufacturer's protocol with the following modifications.
During two-step size selection after adapter ligation, 27.5 \textmu l and 12.5 \textmu l of NEBNext Sample Purification Beads were used in the first and second steps, respectively, to select inserts of 280-290 bp.
Indexing PCR was run at half scale (25 \textmu l total volume) for 3 cycles.
Each PCR product was mixed with 2.5 \textmu l of E-Gel Sample Loading Buffer (Invitrogen) and electrophoresed through a 2\% E-Gel SizeSelect II Agarose Gel (Invitrogen) according to the manufacturer's instructions.
Samples were extracted in 50 \textmu l nuclease-free water (Fisher Bioreagents).
DNA concentrations were measured using a Qubit 3.0 Fluorometer (Thermo Fisher Scientific) according to the manufacturer's protocol.
Samples were pooled and sequenced using an iSeq 100 Sequencing System (Illumina) with a 2 x 150 bp paired-end read length according to the manufacturer's protocol.

\subsection{DMS-MaPseq of live transmissible gastroenteritis virus}


\subsection{Screening coronavirus long-range interactions computationally}
\label{screen_lri_comp}

All coronaviruses with reference genomes in the NCBI Reference Sequence Database~\cite{OLeary2016} were searched for using the following query:
\begin{verbatim}
refseq[filter] AND ("Alphacoronavirus"[Organism] OR
                    "Betacoronavirus"[Organism] OR
                    "Gammacoronavirus"[Organism] OR
                    "Deltacoronavirus"[Organism])
\end{verbatim}
The complete record of every reference genome was downloaded both in FASTA format (for the reference sequence) and in Feature Table format (for feature locations).
The location of the frameshift stimulating element (FSE) in each genome was estimated from the feature table, and the nearest instance of \verb|TTTAAAC| was used as the slippery site, using a custom Python script.
The 2,000 nt segment beginning 100 nt upstream of and ending 1,893 nt downstream of the slippery site was used for predicting long-range interactions involving the FSE.
Genomes with ambiguous nucleotides (e.g. \verb|N|) in this segment were discarded.
For each coronavirus genome, up to 100 secondary structure models of the 2,000 nt segment were generated using Fold version 6.3 from RNAstructure~\cite{Mathews2004a} with \verb|-M 100| and otherwise default parameters.
Then, for each position, the fraction of models for the coronavirus in which the base at the position paired with any other base between positions 101 (the first base of the slippery sequence) and 250 was calculated using a custom Python script.
The coronaviruses were clustered by their fraction vectors using the unweighted pair group method with arithmetic mean (UPGMA) and a euclidean distance metric, implemented in Seaborn version 0.11~\cite{Waskom2021} and SciPy version 1.7~\cite{Virtanen2020}.
The resulting hierarchically-clustered heatmap was examined manually to select coronaviruses based on the prominence of potential long-range interactions with the FSE (relatively large fractions far from positions 101-250).

\end{document}
